### DISCUSSION TITLE
how to cancel an active replace in progress?

### DISCUSSION THREAD

**Discussion by n0xena** [openzfs/zfs#16602] [General]
So as I'm on Arch which just released the 6.11 kernel I built a package for and upgrade. As I also rebuilding a pool after a drive failure I noticed: the 6.11 is not ready yet!
The replace is crawling with 5MB/s and throws some dmesg errors.
Another replace went fine back on 6.10 just two days ago.
So now I want to kill the bad replace in progress - revert back to 6.10 and start over - but there's no option to kill the current resilver - well, other from hard crashing the system.
Am I missing something?

// update
ok - found the issue: I did "kill" the system (enforced un-mount and export) and restarted
after some troubleshooting including checking the smart status I noticed one of the old drives responded rather slow
turns out: shutdown - disconnected it - restartet - now the resilver runs with 800mb/s as it should
tldr: yet another drive failing - good I chose raidz2 - for exactly this double-failure
but request remains: if not yet implemented please add an option to cancel a resilver
